### PR TITLE
Added doc for controlling two or more covers at once

### PR DESCRIPTION
**Pull request by cdce8p**
Unresolved issues:
- open_covers only available when all covers are closed -> Possible fix: allow third state for value_template (half open)
- position value in detailed view are either 0 (closed) or 100 (open), neverthanless set_position works as intended

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/cover.template.markdown [hunk 1]
```diff
@@ -75,3 +75,78 @@ cover:
...
+This example allows you to control two or more covers at once.
+
+```yaml
+cover:
+  - platform: template
+    covers:
+      all_covers:
+        friendly_name: 'All Covers'
+        value_template: >
+          {% if is_state('cover.bedroom', 'open') %}
```
Comment (fabaff) [inline source/_components/cover.template.markdown]: To avoid render of the template, use `{% raw %}{{your_template}}{% endraw %}`.

**Comment by cdce8p** [comment]
Possible fix for issue one (only able to open covers when all are closed): https://github.com/home-assistant/home-assistant-polymer/pull/356

**Comment by cdce8p** [comment]
Hope that looks good now. I'm just figuring out, how to use the `{% raw %}{% endraw %}`.
Take a look at #2948 first and only merge after PR (above) is accepted.

**Comment by frenck** [comment]
@fabaff Could you please check if the last commits solve your concern?

**Pull request merged**